ksqldb-rocksdb-config-setter/src/main/java/io/confluent/ksql/rocksdb/KsqlBoundedMemoryRocksDBConfig.java

```diff
@@ -92,7 +117,34 @@ public class KsqlBoundedMemoryRocksDBConfig extends AbstractConfig {
           Type.DOUBLE,
           INDEX_FILTER_BLOCK_RATIO_DEFAULT,
           Importance.LOW,
-          INDEX_FILTER_BLOCK_RATIO_DOC
+          INDEX_FILTER_BLOCK_RATIO_DOC)
+      .define(
+          COMPACTION_STYLE,
+          Type.STRING,
+          CompactionStyle.UNIVERSAL.name(),
+          ConfigDef.ValidString.in(
+              CompactionStyle.UNIVERSAL.name(), CompactionStyle.LEVEL.name()),
+          Importance.LOW,
+          COMPACTION_STYLE_DOC)
+      .define(
+          COMPRESSION_TYPE,
+          Type.STRING,
+          CompressionType.NO_COMPRESSION.name(),
+          ConfigValidators.enumValues(CompressionType.class),
+          Importance.LOW,
+          COMPRESSION_TYPE_DOC
+      ).define(
+          MAX_BACKGROUND_JOBS,
+          Type.INT,
+          -1,
+          Importance.LOW,
+          MAX_BACKGROUND_JOBS_DOC
+      ).define(
+          COMPACTION_TRIVIAL_MOVE,
```

Comment:
> Out of curiosity, what's the rationale for allowing this config option in particular to be tuned, and not any of the others? (I'm fine with the choice, just curious to understand where it came from.)

This was @cadonna 's suggestion to add this as well so that we enable "COMPACTION_TRIVIAL_MOVE" and if this did not solve the issue, we change the compaction/compression strategies.